packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -75,9 +86,9 @@ export async function createIncomingPaymentService(
   return {
     get: (options) => getIncomingPayment(deps, options),
     create: (options, trx) => createIncomingPayment(deps, options, trx),
-    approve: (id) => approveIncomingPayment(deps, id),
-    cancel: (id) => cancelIncomingPayment(deps, id),
-    complete: (id) => completeIncomingPayment(deps, id),
+    approve: (id, tenantId) => approveIncomingPayment(deps, id, tenantId),
+    cancel: (id, tenantId) => cancelIncomingPayment(deps, id, tenantId),
+    complete: (id, tenantId) => completeIncomingPayment(deps, id, tenantId),
     getWalletAddressPage: (options) => getWalletAddressPage(deps, options),
```

Comment:
We also need to update the `getWalletAddressPage` method so it adds tenantId to the query if provided:

```
  if (tenantId) {
    query.where({ tenantId })
  }
```


In the incoming payment routes, we should also make sure we are passing the tenantId from `ctx.params` into `getWalletAddressPage`